src/debug.h

```diff
@@ -6,8 +6,8 @@
 # define __STDC_FORMAT_MACROS
 #endif
 # include <inttypes.h>
-# define UVWASI_DEBUG(fmt, ...)                                                      \
-    do { fprintf(stderr, fmt, __VA_ARGS__); } while (0)
+# define UVWASI_DEBUG(...)                                                     \
```

Comment:
```suggestion
# define UVWASI_DEBUG(...)                                                           \
```

Reply:
I've updated this now. Thanks! 

Just curious about this specific position of the line continuation character? I changed it to be at position 80 out of habit as I thought it might have been a mistake.

Reply:
I used 79 so that when the line changes, git diffs stay on a single line in my terminal.

Reply:
> I used 79 so that when the line changes, git diffs stay on a single line in my terminal.

That makes sense. Sorry if I'm missing something here, it's still early, but currently it is not at 79 right: https://github.com/nodejs/uvwasi/blob/bb2255ae3c5f7009a846c0053617b1921efcf5db/src/debug.h#L9

Reply:
Yea it looks like the indentation got thrown off [here](https://github.com/nodejs/uvwasi/commit/d3fe61d596873e6b81098ef91a2c881ca9773a4d).